src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -157,11 +160,23 @@ public void doMove(final Map<Territory, ProTerritory> attackMap, final IMoveDele
     ProMoveUtils.calculateAmphibRoutes(player, moveUnits, moveRoutes, transportsToLoad, attackMap, true);
     ProMoveUtils.doMove(moveUnits, moveRoutes, transportsToLoad, moveDel);
 
-    // Calculate attack routes and perform moves
+    // Calculate bombard routes and perform moves
     moveUnits.clear();
     moveRoutes.clear();
     ProMoveUtils.calculateBombardMoveRoutes(player, moveUnits, moveRoutes, attackMap);
     ProMoveUtils.doMove(moveUnits, moveRoutes, null, moveDel);
+
+    // Calculate bombing routes and perform moves
+    moveUnits.clear();
+    moveRoutes.clear();
+    isBombing = true;
```

Comment:
Is the following correct:
- we have a boolean variable that is set to one value on line 172, and another on line 175.
- The same variable is available via a public getter
- We have calls to two static methods, on lines 173 and 174
- `this` object is not passed as a parameter to any of those methods

This either means:
A) The setting of the variable has no effect
B) We are passing in an object to one of static methods, that has a `this` reference, which then uses data in this class, to presumably do something. Ouch :(